lib/psi.rb

```diff
@@ -1448,8 +1448,8 @@ def rsi(construction, film_RSi, temperature = 0.0)
     end
 
     # Opaque materials next.
-    unless m.to_OpaqueMaterial.empty?
-      rsi += m.to_OpaqueMaterial.get.thermalResistance
+    unless m.to_StandardOpaqueMaterial.empty?
+      rsi += m.to_StandardOpaqueMaterial.get.thermalResistance
```

Comment:
Quite the oversight. Without `standard` in there, reported RSi values got ~doubled.